llama_stack/providers/tests/inference/test_inference.py

```diff
@@ -167,7 +168,7 @@ async def test_completion(inference_settings):
     ]
 
     assert all(isinstance(chunk, CompletionResponseStreamChunk) for chunk in chunks)
-    assert len(chunks) == 51
+    assert len(chunks) >= 1
```

Comment:
@ashwinb, this is needed since the number of chunks each provider returns is not consistent and does not depend on the tokens.